sdk/runanywhere-kotlin/docs/DEVELOPMENT.md

```
@@ -0,0 +1,173 @@
+| Mode | Setting | Description |
+|------|---------|-------------|
+| **Local** | `runanywhere.useLocalNatives=true` | Uses JNI libs from `src/main/jniLibs/` (development) |
+| **Remote** | `runanywhere.useLocalNatives=false` | Downloads JNI libs from GitHub releases (consumers) |
+
+The `--setup` flag sets `runanywhere.useLocalNatives=true` automatically.
+
+---
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)